lib/tutorial.js

```diff
@@ -241,10 +244,10 @@ class Tutorial {
     let nextStep;
     if (step) {
       if (typeof step === 'number') {
-        if (step < 0 || step >= this.steps.length) {
+        if (step < 1 || step > this.steps.length) {
```

Comment:
Let's undo this change.  Our current usage of chariot would break, and we already have the `stepNum() / currentStepNum()` methods to surface the one-indexed (human-friendly) index.